src/setupTests.js

```
@@ -41,31 +46,41 @@
+  getAdminSupabaseClient: vi.fn(() => {
+    const chainable = {
+      from: vi.fn().mockReturnValue(chainable),
+      select: vi.fn().mockReturnValue(chainable),
+      insert: vi.fn().mockReturnValue(chainable),
+      update: vi.fn().mockReturnValue(chainable),
+      delete: vi.fn().mockReturnValue(chainable),
+      eq: vi.fn().mockReturnValue(chainable),
+      is: vi.fn().mockReturnValue(chainable),
+      single: vi.fn().mockResolvedValue({ data: null, error: null }),
+      order: vi.fn().mockResolvedValue({ data: [], error: null }),
+      then: vi.fn((resolve) => resolve({ data: [], error: null })),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)